### ISSUE TITLE
Add. 14,738, foll. 114, 115 | https://bl.vuexistapps.us/manuscript/715 (Part22)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1093]
### Problems with Manuscript Overview

- The date in the MS overview is "xith cent." but it should be "about the xith cent."

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- "manu¬script"
- "written in a good, regular hand of about the xith cent." repeats twice. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [X] Typo(s)
- [X] Incorrect content
- [ ] Missing content

### General Notes

- The Additions section is emoty.

**Comment by mikey-tang16** [comment]
Resolved
- Repeated handnote deleted

Unresolved
- Should we add "approximately" to date? If so, do I add directly under history-origin-origDate? 

@wlpotter

**Comment by wlpotter** [comment]
Yes, please add "ca. " before 11th century in the `history/origin/origDate` for this ms part

**Comment by mikey-tang16** [comment]
done!
